easybuild/framework/easyblock.py

```diff
@@ -1708,19 +1832,28 @@ def make_module_step(self, fake=False):
         txt += self.make_module_footer()
 
         mod_filepath = self.module_generator.get_module_filepath(fake=fake)
-        write_file(mod_filepath, txt)
 
-        self.log.info("Module file %s written: %s", mod_filepath, txt)
+        if self.dry_run:
+            if not fake:
```

Comment:
I love the mixture of `dry_run` and `fake`. I can only imagine the look on people's faces when they read this :wink: